PYME/LMVis/importTextDialog.py

```diff
@@ -255,4 +273,33 @@ def GetFieldNames(self):
 
     def GetVarName(self):
         return self.cbVarNames.GetValue()
-        
\ No newline at end of file
+        
+class ImportMatlabDialog(ImportDialog):
+    # MATLAB importer with variable mapping
+    def __init__(self, parent, matFileName):
+        wx.Dialog.__init__(self, parent, title='Import data from MATLAB file')
+
+        self.colNames, self.dataLines = self.MatlabHeaderParse(matFileName)
+
+        ImportDialog.__init__(self, parent)
+
+    def MatlabHeaderParse(self, filename):
+        from scipy.io import loadmat
+        import numpy as np
+
+        mf = loadmat(filename)
```

Comment:
this would be the 3rd time we load the .mat file. Would be sensible to change the function signature to accept either a filename or a loaded file.

```python
def _parse_header(self, file):
    if isinstance(file, six.string_types):
        mf = loadmat(file)
    else:
        #assume file already loaded
        mf = file
````

Reply:
Yep. Changed.